lib/bolt/plugin.rb

```diff
@@ -250,6 +250,44 @@ def by_name(plugin_name)
       end
     end
 
+    # Loads all plugins and returns a map of plugin names to hooks.
+    #
+    def list_plugins
+      load_all_plugins
+
+      hooks = KNOWN_HOOKS.map { |hook| [hook, {}] }.to_h
```

Comment:
Less readable but also less characters: `KNOWN_HOOKS.product([{}]).to_h`

Reply:
I considered using `product` but opted for readability. 😅